src/Uno.Extensions.Core/DispatcherExtensions.cs

```diff
@@ -0,0 +1,48 @@
+﻿namespace Uno.Extensions;
+
+public static class DispatcherExtensions
+{
+	public static Task ExecuteAsync(this IDispatcher dispatcher, Func<CancellationToken, Task> action, CancellationToken token)
```

Comment:
I think we have some standard delefates like FuncAsync or AsyncFunc for the CT/Task signature

Reply:
https://github.com/unoplatform/uno.extensions/blob/main/src/Uno.Extensions.Core/AsyncAction.cs :)